### PR TITLE
add my json

### PR DESCRIPTION
**Pull request by sharad-rajyaguru**

### CODE REVIEW

**Comment by Tweak4141** [comment]
@sharad-rajyaguru Please let your company know that I have sent an email to them. Additionally, please tell them to reply back to me at tweak@is-a-good.dev

File: sub-logs/sharad-rajyaguru.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "owner": {
+    "username": "sharad-rajyaguru",
+    "email": "sharad-rajyaguru@acquaintsofttech.com"
+  },
+  "target": {
+    "CNAME": {
+      "name": "sharad-rajyaguru",
+      "value": "sharad-rajyaguru-dev.github.io"
```
Comment (DEV-DIBSTER) [inline sub-logs/sharad-rajyaguru.json:9]: You need a valid CNAME record. Hit with a 404.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

**Review by DEV-DIBSTER — APPROVED** [review]

**Comment by Tweak4141** [comment]
@sharad-rajyaguru Please star the repository.

**Comment by wclarkey** [comment]
@sharad-rajyaguru If the repository is not starred in 1-3 business days (GMT) this and the other pull requests will be closed.

**Comment by sharad-rajyaguru** [comment]
> ```json
> sharad-rajyaguru.github.io
> ```

@Tweak4141 starred this repository

**Review by Tweak4141 — APPROVED** [review]

**Pull request merged**